README.rst

```diff
@@ -15,7 +15,7 @@ ctapipe |ci| |codacy| |coverage| |conda| |doilatest|
   :target: https://zenodo.org/badge/latestdoi/37927055
 
 Low-level data processing pipeline software for
-`CTA <www.cta-observatory.org>`_ (the Cherenkov Telescope Array)
+`CTA <www.cta-observatory.org>`__ (the Cherenkov Telescope Array)
```

Comment:
what does the double underscore mean? I don't see that in any RST documentation (only single underscore for a link )